### PR TITLE
feat: Add TrajectoryPrinter helper algorithm

### PR DESCRIPTION
**Pull request by paulgessinger**
This PR adds a helper algorithm that prints out Trajectories. I'm using this to debug. It might not be useful for a long time, but I thought it might make sense to add it at this point in time anyway.

### CODE REVIEW

File: Examples/Python/python/acts/examples/reconstruction.py [hunk 1]
```diff
@@ -881,6 +881,13 @@ def addCKFTracks(
 
     s.addWhiteboardAlias("trajectories", trackFinder.config.outputTrajectories)
 
+    s.addAlgorithm(
+        acts.examples.TrajectoryPrinter(
+            level=customLogLevel(),
+            inputTrajectories=trackFinder.config.outputTrajectories,
+        )
+    )
```
Comment (andiwand) [inline Examples/Python/python/acts/examples/reconstruction.py:889]: should we hide this behind a flag or just have it switching on with the correct debug level?

**Review by andiwand — APPROVED** [review]
looks good just left one question

File: Examples/Python/python/acts/examples/reconstruction.py (hunk 1 above)
Comment (paulgessinger) [reply Examples/Python/python/acts/examples/reconstruction.py:889]: Hm, could do I guess.

**Comment by paulgessinger** [comment]
On second thought, maybe we don't actually need to put this in.

**Pull request closed without merging**